mkdocs.yml

```diff
@@ -1,4 +1,4 @@
-site_name: MLPerf ABTF Inference Documentation
+site_name: "MLPerf Inference: Automotive PoC Documentation"
```

Comment:
Can you rename to:
- MLPerf Automotive POC Documentation

Reply:
Done 👍